src/search/FindInFiles.js

```diff
@@ -830,6 +856,56 @@ define(function (require, exports, module) {
             }
         });
     };
+    
+    /**
+     * This stores file system events emitted by watchers that were not yet processed
+     */
+    var _cachedFileSystemEvents = [];
+    
+    /**
+     * Debounced function to process emitted file system events
+     * for cases when there's a lot of fs events emitted in a very short period of time
+     */
+    _processCachedFileSystemEvents = _.debounce(function () {
+        // we need to reduce _cachedFileSystemEvents not to contain duplicates!
+        _cachedFileSystemEvents = _cachedFileSystemEvents.reduce(function (result, obj) {
+            var fullPath = obj.entry ? obj.entry.fullPath : null;
```

Comment:
Deeper in the code there's a handle for this case (doing a full refresh I think) so it didn't want to stop it here.